tests/FactoryTest.php

```diff
@@ -2,8 +2,8 @@
 
 namespace PUGX\Shortid\Test;
 
-use PHPUnit\Framework\Assert;
 use PHPUnit\Framework\TestCase;
+use RandomLib\Factory as RandomLibFactory;
```

Comment:
this is violating `ordered_imports` cs rule.
I guess that you failed to run `php-cs-fixer fix -v`

Reply:
Hi @garak, thanks for your reply.

I've fixed this via `PHP-CS-Fixer` tool :).